.github/workflows/maven.yml

```diff
@@ -0,0 +1,23 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        java: [8, 8.0.192, 11, 11.0.2, 13, 13.0.4, 15, 16-ea]
+      fail-fast: false
+      max-parallel: 2          
+    name: Test JDK ${{ matrix.java }}, ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+    - name: Test with Maven
+      run: mvn test -B --file pom.xml
```

Comment:
According to the documentation is recommended to use `mvn -B --file pom.xml test` I suggest you `mvn -B --file pom.xml verify`

Reply:
Done, below.